### PR TITLE
fix(docs): add 2020 goals to roadmap

### PR DESCRIPTION
**Pull request by dhmlau**
Add 2020 goals and focus to the roadmap.

cc @bajtos @raymondfeng , per our conversation. 

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

**Comment by dhmlau** [comment]
@agnes512 @bajtos @raymondfeng, originally I was thinking that once we have this approved, we can put this content to the Q4 summaries. But I start to think whether it's a good idea to have a separate blog.

File: docs/ROADMAP.md [hunk 1]
```diff
@@ -1,3 +1,38 @@
+## 2020 Goals and Focus
+
+### Goal#1 - Ensure our framework and docs are ready for LB3 users to migrate to LB4
```
Comment (bajtos) [inline docs/ROADMAP.md]: I am proposing to set a more ambitious goal:

```suggestion
### Goal#1 - Enable as many LoopBack 3 users to migrate to LoopBack 4 as possible
```

Thoughts?

File: docs/ROADMAP.md [hunk 2]
```diff
@@ -1,3 +1,38 @@
...
+
+With LoopBack 3 going end-of-life at the end of 2020, we are continuing to focus
+on the following areas:
+
+- provide a migration guide to help LB3 users to migrate their applications to
```
Comment (bajtos) [inline docs/ROADMAP.md]: A proposal for consideration, feel free to ignore.

```suggestion
- finish the migration guide to help LB3 users to migrate their applications to
```

File: docs/ROADMAP.md [hunk 3]
```diff
@@ -1,3 +1,38 @@
...
+  LB4
+- offer some level of tooling support to make the migration is easier
+- fill in feature parity gaps by either implementing the features in LB4 or
+  recommending a solution using existing libraries
+- add APIC/LB4 integration
```
Comment (bajtos) [inline docs/ROADMAP.md]: Let's replace acronyms with full names here, I am concerned this line can be too cryptic for people outside of our team.

```suggestion
- Integrate LoopBack 4 into IBM API Connect 
```

File: docs/ROADMAP.md [hunk 4]
```diff
@@ -1,3 +1,38 @@
...
+
+### Goal#2 - Keep LoopBack staying relevant regarding latest technologies
```
Comment (bajtos) [inline docs/ROADMAP.md]: Nitpick - feel free to ignore

```suggestion
### Goal#2 - Keep adopting latest technologies to stay relevant
```

File: docs/ROADMAP.md [hunk 5]
```diff
@@ -1,3 +1,38 @@
...
+- modernize the connector infrastructure and code base
+
+### Goal#3: Nurture our contribution community to encourage more contributions
+
+Community contributions are vital to us. It not only accelerates our
+development, but also helps building deep LoopBack knowledge in the community.
+
+We would like to encourage community contributions by make available to the
+community a list of GH issues that we want to finish in a short term by using
+the `help wanted` and `2020x` labels.
```
Comment (bajtos) [inline docs/ROADMAP.md]:
```suggestion
the `help wanted` and `2020Qx` labels.
```

File: docs/ROADMAP.md [hunk 6]
```diff
@@ -1,3 +1,38 @@
...
+possibility of using and/or integrating latest technologies with LoopBack 4.
+
+Some of the areas we'd like to explore, but not limited to:
+
+- messaging and pub/sub event style, e.g. Kafka, MQTT
+- cloud native integration
+- native support on GraphQL
+- modernize the connector infrastructure and code base
+
+### Goal#3: Nurture our contribution community to encourage more contributions
```
Comment (bajtos) [inline docs/ROADMAP.md]:
```suggestion
### Goal#3: Nurture our community and encourage more contributions
```

**Comment by dhmlau** [comment]
Thanks @bajtos  for your suggestion. I have accepted all your suggested changes.

File: docs/ROADMAP.md [hunk 7]
```diff
@@ -1,3 +1,38 @@
+## 2020 Goals and Focus
+
+### Goal#1 - Enable as many LoopBack 3 users to migrate to LoopBack 4 as possible
+
+With LoopBack 3 going end-of-life at the end of 2020, we are continuing to focus
+on the following areas:
+
+- finish the migration guide to help LB3 users to migrate their applications to
+  LB4
+- offer some level of tooling support to make the migration is easier
```
Comment (jannyHou) [inline docs/ROADMAP.md]: nitpick: extra `is`

**Pull request merged**